snapcraft.yaml

```diff
@@ -198,9 +201,14 @@ parts:
       MACH="env -u LD_LIBRARY_PATH -u PYTHONPATH /usr/bin/python3 ./mach"
       $MACH configure --prefix=$SNAPCRAFT_PART_INSTALL/usr
       $MACH build -j$SNAPCRAFT_PARALLEL_BUILD_COUNT
+      export DUMP_SYMS=$(pwd)/dump_syms/dump_syms
+      $MACH buildsymbols
       $MACH install
       DISTRIBUTION=$SNAPCRAFT_PART_INSTALL/usr/lib/firefox/distribution
       mkdir -p $DISTRIBUTION/extensions
+      if [ ! -f "$SNAPCRAFT_PROJECT_DIR/symbols-upload-token" ]; then
```

Comment:
Not sure I understand this if block. If we don't have an upload token, we ship the symbols in the snap, is that right? It might be worth a comment to explain the rationale.

Reply:
This is right, but this was before I got news from the launchpad meeting. This can go away.